scripts/update_config.py

```diff
@@ -0,0 +1,24 @@
+'''
+Overwrite a config file by renaming args.
+Require one argument: path_to_file.
+'''
+
+import sys
+
+# Mapping - key: old name, value: new name
+name_dict = {'task_patience':'lr_patience',\
+               'do_train': 'do_pretrain',\
+               'train_for_eval':'do_target_task_training',\
+               'do_eval': 'do_full_eval',\
+               'train_tasks':'pretrain_tasks',\
+               'eval_tasks':'target_tasks'}
+
+path = sys.argv[1]
+assert '.conf' in path, "Error: .conf not found."
+with open(path, 'r+') as f:
+    param = f.read()
+    for old_name, new_name in name_dict.items():
+        # avoid substrings, won't replace first line
+        param = param.replace('\n' + old_name, '\n' + new_name)
```

Comment:
This is somewhat dangerous since it assumes well-behaved line breaks, and that the names aren't found in any strings values (like file paths).

It would be safer to use the utils in `src/config.py`, in particular `params_from_file()` and `write_params()`; you could do something like:
```
params = params_from_file(path)
for old_name, new_name in name_dict.items():
    if old_name in params:
        params[new_name] = params[old_name]
        del params[old_name]
write_params(params, path)
```

Reply:
(FYI, I just added support for `del params[key]` to the Params object)